book/src/running-validator/validator-stake.md

```diff
@@ -33,13 +43,56 @@ solana redeem-vote-credits ~/validator-stake-keypair.json ~/validator-vote-keypa
 
 The rewards lamports earned are split between your stake account and the vote account according to the commission rate set in the vote account. Rewards can only be earned while the validator is up and running. Further, once staked, the validator becomes an important part of the network. In order to safely remove a validator from the network, first deactivate its stake.
 
-Stake can be deactivated by running:
+At the end of each slot, a validator is expected to send a vote transaction. These vote transactions are paid for by lamports from a validator's identity account.
+
+This is a normal transaction so the standard transaction fee will apply. The transaction fee range is defined by the genesis block. The actual fee will fluctuate based on transaction load. You can determine the current fee via the [RPC API “getRecentBlockhash”](https://solana-labs.github.io/book-edge/jsonrpc-api.html#getrecentblockhash) before submitting a transaction.
```

Comment:
Can you update this getRecentBlockhash link to be to docs.solana.com?
(Might need updating in TdS docs too...)